python/ray/util/placement_group.py

```diff
@@ -147,6 +148,12 @@ def placement_group(
             will fate share with its creator and will be deleted once its
             creator is dead, or "detached", which means the placement group
             will live as a global object independent of the creator.
+        max_cpu_fraction_per_node: The maximum fraction of CPU cores this placement
+            group can take up on each node. This must be a float between 0 and 1. If
+            provided, the placement group won't take up more than
+            max_cpu_fraction_per_node * node["num_cpus"] CPU cores on each node. This
+            is useful for ensuring that some percentage of CPU cores are available on
+            each node for workloads that aren't using this placement group.
```

Comment:
Can we also document / guarantee that any non-zero value here will reserve at least 1 CPU per node?

Reply:
Do you think it's also reasonable to always round up? (e.g., 5 CPUs , 50% fraction -> can reserve up to 3 CPUs)